framework/src/modules/chain/components/storage/entities/block_temp.js

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const path = require('path');
+const assert = require('assert');
+const { defaults, pick } = require('lodash');
+const {
+	entities: { BaseEntity },
+	utils: {
+		filterTypes: { NUMBER, TEXT },
+	},
+} = require('../../../../../components/storage');
+
+const sqlFiles = {
+	create: 'block_temp/create.sql',
+	delete: 'block_temp/delete.sql',
+	get: 'block_temp/get.sql',
+};
+
+/**
+ * Block temp
+ * @typedef {Object} block_temp
+ * @property {string} id
+ * @property {number} height
+ * @property {json} block
```

Comment:
```suggestion
 * @property {json} fullBlock
```

Reply:
I think `json` should be `object` instead.